core/arena/stringify.js

```diff
@@ -0,0 +1,21 @@
+import { isNull } from './isNull'
+import { isEmpty } from './isEmpty'
+
+import { mapToObject } from './mapToObject'
+
+/**
+ * Stringify any object.
```

Comment:
this should be Converts a JavaScript value to a JavaScript Object Notation (JSON) string.